tests/acceptance/bootstrap/ArchiverContext.php

```diff
@@ -325,4 +325,27 @@ public function theDownloadedArchiveShouldContainTheseFiles(string $type, TableN
 		\unlink($tempFile);
 		$this->removeDir($tempExtractFolder);
 	}
+
+	/**
+	 * @Then user :user should be able to download archive of federated shared folder :resource
+	 *
+	 * @param string $user
+	 * @param string $resource
+	 *
+	 * @return void
+	 */
+	public function userShouldBeAbleToDownloadArchiveOfFederatedSharedFolder(string $user, string $resource): void {
+		$queryString = $this->getArchiverQueryString($user, $resource, 'remoteItemIds');
```

Comment:
i had seen use of `$this->archiverContext->getArchiverUrl` in `tests/acceptance/bootstrap/SpacesContext.php`.
So, confused about the perfect place to add this step definition 
@saw-jan @nirajacharya2 what do you think about where should we add a related step- definition if that step is part of space context also. 
```
             * @When /^public downloads the folder "([^"]*)" from the last created public link using the public files API$/
	 * @When user :user tries to download the space :spaceName owned by user :owner using the WebDAV API
	 * @When /^user "([^"]*)" (?:downloads|tries to download) the space "([^"]*)" using the WebDAV API$/
```

Reply:
this is fine.